DEVELOPMENT.md

```diff
@@ -94,3 +94,14 @@ Then `package.json` should have the same versions specified as well:
 ```
 
 If you have a mismatch in `package.json`, say `fhir-works-on-aws-routing` was set to `4.1.0` instead of `4.0.0`. An error message
+
+### Accessing Logs and Debugging on FWoA
+Logs and Debugging are handled by [CloudWatch](https://docs.aws.amazon.com/cloudwatch/index.html). Detailed instructions on customizing CloudWatch to organize troubleshooting can be found [here](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/WhatIsCloudWatch.html).
+
+To access the logs generated by FWoA:
+1. Log on to the AWS Console.
+2. From the list of AWS services, select **CloudWatch**.
+3. From the tab on the left, select **Log Groups**
+4. From the list of log groups, select the Lambda Function or the APIGateway service that you would like to debug.
```

Comment:
Can we add some sub-bullet points with brief introduction on what each Lambda and APIGateway log can be used for?

Reply:
That makes sense, I'll write out what each Log Group tracks and will update this PR